src/content/docs/kubernetes-pixie/kubernetes-integration/installation/kubernetes-integration-install-configure.mdx

```diff
@@ -32,54 +32,36 @@ import kubernetesCke from 'images/kubernetes_logo_cke.png'
 
 import kubernetesEks from 'images/kubernetes_logo_eks.png'
 
-The easiest way to install the Kubernetes integration is to use our guided install to generate a manifest. It bundles not just the integration DaemonSets, but also other New Relic Kubernetes configurations, like [Kubernetes events](/docs/integrations/kubernetes-integration/kubernetes-events/install-kubernetes-events-integration), [Prometheus OpenMetrics](/docs/integrations/prometheus-integrations/get-started/new-relic-prometheus-openmetrics-integration-kubernetes), and [New Relic log monitoring](/docs/logs).
+The easiest way to install the Kubernetes integration is to use our guided install to generate a manifest. It bundles not just the integration DaemonSets, but also other New Relic Kubernetes configurations, like [Kubernetes events](/docs/integrations/kubernetes-integration/kubernetes-events/install-kubernetes-events-integration), [Prometheus Agent](/docs/infrastructure/prometheus-integrations/install-configure-prometheus-agent/install-prometheus-agent/), and [New Relic log monitoring](/docs/logs).
 
-As a Kubernetes user, you might consider [Auto-telemetry with Pixie](/docs/auto-telemetry-pixie/get-started-auto-telemetry-pixie) for deeper insights into your Kubernetes clusters and workloads. No language agents required.
-
-Ready to get started? If you don't already have one, [sign up for a New Relic account](https://newrelic.com/signup). It's free, forever!
+You have these options to install the Kubernetes integration:
 
-## Use guided install [#installer]
+* Downloading the [New Relic Kubernetes quickstart](https://newrelic.com/instant-observability/kubernetes). The quickstart includes pre-built resources like dashboards, instrumentations, and alerts.
```

Comment:
I am not familiar with the quickstart, but makes me wonder (as a user) whether I need to do both things. Will the quickstart guide me through the installation? Do I need to do it before following the steps on this page?

Reply:
I've removed it because @csongnr agrees with your comment.